src/OpenLoco/src/Ui/Chart.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "Graphics/Colour.h"
+#include "Types.hpp"
+#include <OpenLoco/Core/EnumFlags.hpp>
+#include <cstddef>
+
+namespace OpenLoco::Gfx
+{
+    class DrawingContext;
+}
+
+namespace OpenLoco::Ui
+{
+    struct Window;
+
+    enum class GraphFlags : uint32_t
+    {
+        none = 0U,
+        showNegativeValues = 1U << 0,
+        dataFrontToBack = 1U << 1,
+        hideAxesAndLabels = 1U << 2,
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(GraphFlags);
+
+#pragma pack(push, 1)
```

Comment:
Do we still need to pack?

Reply:
We won't if we let go of the loco_global, as you suggested. Great! Also means the enum flags no longer need to be uint32_t.